packages/lambda-tiler/src/cli/serve.ts

```diff
@@ -58,6 +58,10 @@ function useAws(): void {
             } as any,
             logger,
         );
+        const ifNoneMatch = req.header(HttpHeader.IfNoneMatch);
+        if (ifNoneMatch != null) {
+            ctx.headers.set(HttpHeader.IfNoneMatch.toLowerCase(), ifNoneMatch);
+        }
```

Comment:
Maybe I should set all the headers?

Reply:
If you want improve the CLI go for it :)

Reply:
maybe later :)